netci.groovy

```diff
@@ -151,3 +151,20 @@ set TMP=%TEMP%
   Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto')
   addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
 }
+
+// Perf Correctness
+commitPullList.each { isPr ->
+  def jobName = Utilities.getFullJobName(projectName, "perf_correctness", isPr)
+  def myJob = job(jobName) {
+    description('perf test correctness')
+    label('windows-roslyn')
+    steps {
+      batchFile(""".\\cibuild.cmd /testPerfCorrectness""")
+    }
+  }
+
+  def triggerPhraseOnly = true
```

Comment:
This should eventually be `false`.  When `true` it will only run when developers specifically ask for it in PR.